config/routes.rb

```diff
@@ -7,3 +7,13 @@
 
   root to: "categories#index"
 end
+
+Rails.application.routes.draw do
+  namespace :api, defaults: { format: "json" } do
+    namespace :v1 do
+      namespace :domains do
+        post "scan", to: "scans#create"
```

Comment:
We can, but this is an action, not just retrieving data, so I decided to make post